static/webpack.config.js

```diff
@@ -33,6 +33,10 @@ const config = {
       __dirname + "/js/place/place.ts",
       __dirname + "/css/place/place.scss",
     ],
+    choropleth: [
```

Comment:
Please keep this list sorted, and move the code to the tools subdirectory.

Reply:
Moved